modules/tensor_mechanics/doc/content/documentation/systems/ScalarKernels/GeneralizedPlaneStrain.md

```diff
@@ -1,15 +1,16 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # GeneralizedPlaneStrain
 
-!alert construction title=Undocumented Class
-The GeneralizedPlaneStrain has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
-
 !syntax description /ScalarKernels/GeneralizedPlaneStrain
 
+ScalarKernel `GeneralizedPlaneStrain` assembles components of the residual and diagonal jacobian corresponding to a given scalar variable. The values of the residual and diagonal jacobian components are fetched from the [GeneralizedPlaneStrainUserObject](/GeneralizedPlaneStrainUserObject.md).
+
+The equilibrium condition condition in the out-of-plane direction is given as
+\begin{equation}
+	\int_{A}{\sigma_{zz}dA} = \bar{N}_{zz}
+\end{equation}
+
+A detailed description of generalized plane strain formulation can be found in [here](tensor_mechanics/generalized_plane_strain.md).
+
```

Comment:
Mention here that this is typically set up by the Action, and add a link to that documentation.

Reply:
I second @bwspenc's comment.  There's a set phrase in an note alert that I've used in Bison that you could use here too (grep for Often Created)